### DISCUSSION TITLE
Add Mac Support

### DISCUSSION THREAD

**Discussion by StoppedwummSites** [project64/project64#2396] [Ideas]
Big Emulators like Dolphin have Apple Support, maybe it would be possible to emulate on Mac via the Metal API.

**Comment by flagrama** [comment]
Multi-platform support is planned for the future. Only fixing the Android version and Linux support is planned currently, likely because building for Mac is expensive. If someone else wants to maintain and support a fork with a mac build after that is done, then that may need to be the path taken. It's possible in the far future Zilmar may decide to maintain a build himself, but there just aren't any plans currently.

**Comment by project64** [comment]
I am in the process of getting android working, hopefully getting android complete gives me 80-90% of the work done for linux, and getting linux done gives me 95% of a mac version.

As @flagrama said developing on mac is expensive and they have no love for emulation, so it is at the bottom of my list of systems to look at
